src/main/java/org/opentripplanner/routing/algorithm/raptoradapter/transit/cost/CostCalculatorFactory.java

```diff
@@ -6,11 +6,11 @@ public class CostCalculatorFactory {
 
   public static <T extends DefaultTripSchedule> RaptorCostCalculator<T> createCostCalculator(
     GeneralizedCostParameters generalizedCostParameters,
-    int[] stopBoardAlightCosts
+    int[] stopTransferCosts
```

Comment:
My reasoning was that it looks like the costCalculator treats the boardAlightCosts as transferCosts, and thus it would be a little bit clearer what was going on if this was reflected in the signature for the createCostCalculator() method.

But as discussed in the meeting it might be better to draw the line at the module border. I'm fine with reverting this.

Reply:
Sorry for the confution here. This is the case:

- Raptor does not have transferCost only alight and board cost, so the transfer cost is splitt and added either to boarding(if symetrical) or in this case to both alight and boarding - to stops may have different cost - hence we need to add a cost for both alighting and boarding. 

This "mess" should probably be visited again and cleaned up - the cost model have grown and it might be just a bit more clear if Raptor supported board, alight and transfer costs - unsure what the performance overhead it. Performance was the original reason for why this is like it is. Compressing stop information in memory increases the performance.

So this feature comes from NeTEX where the name is transfer, but since we are adding the cost to alight and boarding (both ends) I think the name in the OTP model i wrong - the mapping should happen when going from stopPriority to cost. A few comments in the model would have helped as well.

In `TransitRoutingConfig` and `TransitTuningParameters` this should be renamed to something like `boardAndAlightCostForStopTransferPriority` - a shorter name and a better java doc is probably the best solution.

Reply:
Sumarized: boardAndAlight cost is added twice (both ends of a transfer), while transferCost is added once.

Reply:
Thanks for that explanation!

The confusing thing about the name "stopBoardAlightCosts" is that it sounds like it should be applied for the first boarding and the last alighting as well. There is nothing that indicates that it is specific to transfers. But calling it stopBoardAlightTransferCosts may be a bit long?

Here are some possible alternatives for the name of this concept:
- stopTransferCosts
- stopBoardAlightCosts
- stopBoardAlightTransferCosts
- stopBoardAlightDuringTransferCosts
- stopHalfTransferCosts

Do you think we should go with "stopBoardAlightCosts"?

I'm ok with that and think the naming is perhaps less important than being consistent, so I'm happy to make the naming consistent and will add some javadoc.

Reply:
`stopBoardAlightTransferCosts` with JavaDoc